vendor/qcrilam_app.te

```diff
@@ -15,7 +15,8 @@ allow qcrilam_app audioserver_service:service_manager find;
 allow qcrilam_app radio_service:service_manager find;
 
 # Find the vendor.qti.hardware.radio.am::IQcRilAudio HIDL service
-allow qcrilam_app vnd_qcrilhook_hwservice:hwservice_manager find;
+# allow qcrilam_app vnd_qcrilhook_hwservice:hwservice_manager find;
```

Comment:
remove instead of commenting lines

Reply:
> Note that this PR is draft and request for comments; changes and commits have to be cleaned up

It's right in the PR description :smirk: :wink: